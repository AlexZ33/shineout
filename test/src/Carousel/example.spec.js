import exampleTest from '../../example'

exampleTest('Carousel')
