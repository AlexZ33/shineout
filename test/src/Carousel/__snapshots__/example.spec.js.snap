// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Carousel[site/pages/components/Carousel/example-1-base.js] 1`] = `
<div>
  <div>
    animation:
     
    <div>
      <div>
        <div>
          <span>
            slide
          </span>
          <a />
        </div>
        <div>
          <span>
            Data not found
          </span>
        </div>
      </div>
    </div>
    　indicatorPosition: 
    <div>
      <div>
        <div>
          <span>
            center
          </span>
          <a />
        </div>
        <div>
          <span>
            Data not found
          </span>
        </div>
      </div>
    </div>
    　indicatorType: 
    <div>
      <div>
        <div>
          <span>
            circle
          </span>
          <a />
        </div>
        <div>
          <span>
            Data not found
          </span>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div>
      <div>
        <div>
          Page 1
        </div>
      </div>
    </div>
    <div>
      <div>
        <div>
          Page 2
        </div>
      </div>
    </div>
    <div>
      <div>
        <div>
          Page 3
        </div>
      </div>
    </div>
    <div>
      <a>
        <img />
      </a>
    </div>
    <div>
      <a>
        
      </a>
      <a>
        
      </a>
      <a>
        
      </a>
      <a>
        
      </a>
    </div>
  </div>
</div>
`;
